conf/zapiperf/cdot/9.8.0/disk.yaml

```diff
@@ -36,7 +36,7 @@ plugins:
   # any names after the object names will be threated as
   # label names that will be added to instances
     - node
-    - aggr node
+    - aggr ...
```

Comment:
is ... correct?

Reply:
Yes. 
So we are aggregating data at node and aggr level and all metrics like user_read_chain and disk_busy would be available at node/aggr not at disk. Also talked with Vachgan and get to know that getting at disk level may impact performance as more calculations would be involved.

But we would need info of plex and raid where we can show how many plex,raid would be in system, so metric calculation wise it's same just with this aggr_xxx_xxx metric would be showing other labels as well along with node,cluster,datacenter.

Ex: for raid count:                    
count( count by(raid) (aggr_disk_busy{datacenter="$Datacenter",cluster="$Cluster",node=~"$Node"}) )

Reply:
just to add, flashcache and flashpool charts are pending as my cluster don't have them, doing in next PR.

Reply:
thanks - got it, so you're wanting to copy all labels of the original instance